### PR TITLE
Added content to README.md

### PR DESCRIPTION
**Pull request by Satanaelcode**
~ Changes
+ Xanthe Enterprise
+ IsThisARat

### CODE REVIEW

File: docs/malware-scanner/README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-## *Currently empty, check back later or contribute yourself*
\ No newline at end of file
+## General Malware Scanner
+[IsThisARat?](https://isthisarat.com/)
+
+Is an Rat finder powerd by AI
```
Comment (cn-cn-cn) [inline docs/malware-scanner/README.md:4]:
```suggestion
A website that attempts to detect [RAT](https://en.wikipedia.org/wiki/Remote_desktop_software#Malicious_variants)s using AI.
```

File: docs/malware-scanner/README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+## Plugin Malware Scanner
+[ByteScan - Xanthe Enterprise](https://discord.gg/NNeZjUjKwG)
+
+Is an great scanner for plugins because of it getting general updated and fast support 
+are there no problems
```
Comment (cn-cn-cn) [inline docs/malware-scanner/README.md:9]: As written, this seems like advertising. Also, the paid nature of the plugin is not mentioned. Please rewrite or remove this.

**Review by notcancername — CHANGES_REQUESTED** [review]

File: docs/malware-scanner/README.md (hunk 2 above)
Comment (RayBytes) [inline docs/malware-scanner/README.md:9]: I would also recommend to remove this. This just seems like pure advertising.

File: docs/malware-scanner/README.md (hunk 1 above)
Comment (RayBytes) [inline docs/malware-scanner/README.md:4]: I don't think it's wise to recommend relatively unknown websites to use. MMPA is already working on its own malware scanner, that can be linked when its in a working state.
The scanner may also false, with no one in the MMPA having any control over. Checking their discord right now shows that anything which may be obfuscated seems to be flagged as a rat.

**Review by RayBytes — COMMENTED** [review]
All in all; I do not think this pull request is at all necessary.

File: docs/malware-scanner/README.md (hunk 1 above)
Comment (cn-cn-cn) [reply docs/malware-scanner/README.md:4]: I agree. Obfuscation [as detected here](https://isthisarat.com/score/d82f176a5877dd941ad2f8586becd00ef0bed75be75b7f029f6b649ec67f4a45) is unfortunately very common among Minecraft plugins. Like the other plugin, it's also proprietary.

In my personal opinion, this should not be recommended by this project.

Comment (RayBytes) [reply docs/malware-scanner/README.md:4]: > I agree. Obfuscation [as detected here](https://isthisarat.com/score/d82f176a5877dd941ad2f8586becd00ef0bed75be75b7f029f6b649ec67f4a45) is unfortunately very common among Minecraft plugins. Like the other plugin, it's also proprietary.
> 
> In my personal opinion, this should not be recommended by this project.

Agreed 👍

**Comment by xyzeva** [comment]
Not needed, please see [concoction](https://github.com/Minecraft-Malware-Prevention-Alliance/concoction)

**Comment by SylvKT** [comment]
get spammed on

**Pull request closed without merging**